src/tools/clippy/clippy_lints/src/future_not_send.rs

```diff
@@ -99,7 +99,7 @@ impl<'tcx> LateLintPass<'tcx> for FutureNotSend {
                                     db.note(format!(
                                         "`{}` doesn't implement `{}`",
                                         trait_pred.self_ty(),
-                                        trait_pred.trait_ref.print_only_trait_path(),
+                                        trait_pred.trait_ref.print_trait_sugared(),
```

Comment:
I guess this doesn't matter -- can revert. I actually don't even know why this uses `print_only_trait_path`, given that it's always `Send`.

Reply:
It's likely copy-pasted from a more general rustc error.